pkg/apis/autoscaling/register.go

```diff
@@ -30,7 +30,17 @@ const (
 	MinScaleAnnotationKey = GroupName + "/minScale"
 	MaxScaleAnnotationKey = GroupName + "/maxScale"
 
+	// HPA is the value of ClassAnnotationKey for PodAutoscaler of HPA.
+	HPA = "hpa"
+	// KPA is the value of ClassAnnotationKey for PodAutoscaler of KPA.
+	KPA = "kpa"
```

Comment:
These should use this domain style: https://github.com/knative/serving/blob/2a6b811207f2fa4055b115322a0f5d66c5830eb5/pkg/reconciler/v1alpha1/clusteringress/clusteringress.go#L46